### PR TITLE
Performance Optimization. Reduce CPU usage from 100ms per sec to 10 ms per sec

### PR DESCRIPTION
**Pull request by euloanty**
Thank you!

### CODE REVIEW

File: Modules/QuestieCoordinates.lua [hunk 1]
```diff
@@ -88,6 +88,7 @@ function QuestieCoords:Initialize()
 end
 
 function QuestieCoords:Update(elapsed)
+	Questie:SendMessage("QUESITE_ICON_ON_UPDATE")
```
Comment (BreakBB) [inline Modules/QuestieCoordinates.lua]:
```suggestion
	Questie:SendMessage("QUESTIE_ICON_ON_UPDATE")
```

File: Modules/QuestieNameplate.lua [hunk 2]
```diff
@@ -5,14 +5,9 @@ local npFrames = {};
...
 
-QuestieNameplate.GlobalFrame = nil;
-
 -- Initializer
 function QuestieNameplate:Initialize()
-    if QuestieNameplate.GlobalFrame == nil then
-        QuestieNameplate.GlobalFrame = CreateFrame("Frame");
-        QuestieNameplate.GlobalFrame:SetScript("OnUpdate", QuestieNameplate.UpdateNameplate);
-    end
+	Questie:RegisterMessage("QUESITE_ICON_ON_UPDATE",QuestieNameplate.UpdateNameplate)
```
Comment (BreakBB) [inline Modules/QuestieNameplate.lua]:
```suggestion
	Questie:RegisterMessage("QUESTIE_ICON_ON_UPDATE",QuestieNameplate.UpdateNameplate)
```

Comment (euloanty) [reply Modules/QuestieNameplate.lua]: No GlobalFrame in the code anymore. this needs to be removed.

**Comment by Logonz** [comment]
Hey!
This is already fixed in the v4.1.2 branch.
Also we don't really want to put a message from QuestieComms -> QuestieNameplates as that would create a dependency between them.

Thanks for the interest!

**Pull request closed without merging**